apps/ai-image-generator/backend/src/transform-images.ts

```diff
@@ -128,7 +180,32 @@ export class ImageTransformer {
         height: info.height,
         channels: 4,
       },
-    }).toFormat('png');
+    });
+  }
+
+  private toDimensions(width: number | undefined, height: number | undefined): Dimensions {
+    if (typeof width === 'undefined' || typeof height === 'undefined') {
+      throw new Error('no width or height');
+    }
+    const ratio = width / height;
+    let layout: Dimensions['layout'];
+
+    if (ratio > 1) {
+      layout = 'landscape';
+    } else if (ratio < 1) {
```

Comment:
Not sure if there's ever a reality where this happens, but technically negative numbers would pass here even though it shouldn't. Maybe part of the `Math itself is broken` error should be to check for negative widths and heights?